docs/source/quick_start/getting_start_with_tensorbay.rst

```diff
@@ -10,109 +10,78 @@ To install TensorBay SDK and CLI by **pip**, run the following command:
 
```

Comment:
About the commit message,
This commit involves not only `examples` scope, I think we can remove it.
Also, the message `refactor xxx chapter` is not clear enough. You need to describe your changes more detailedly.